### PR TITLE
Adding comma in DocString

### PR DESCRIPTION
**Pull request by athirai-s**
**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `corobo mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by abhishalya** [comment]
Please squash the commits into a single commit. Also the PR title seems to be off.

**Comment by athirai-s** [comment]
Done!

**Comment by abhishalya** [comment]
Well you added a new commit and now it has conflicts too. I just want a single commit.

**Comment by abhishalya** [comment]
This does not look good, please ask for help on gitter if you need help using git :)

File: docs/Developers/Git_Basics.rst [hunk 1]
```diff
@@ -329,7 +329,7 @@ to understand how it works.
 Rebasing
 --------
 
-As people work on coala new commits will be added. This will result in your
+As people work on coala, new commits will be added. This will result in your
 local fork going out of sync with the remote repository.
 To sync your changes with the remote repository run the following commands in
 the desired branch:
```
Comment (utkarsh2102) [inline docs/Developers/Git_Basics.rst:335]: Actually, you're editing the very doc that you need to use 😄 
Maybe follow this very doc and see if you can come up with something?
`git rebase -i [...]` is your friend, let me know if you need help!

**Review by utkarsh2102 — CHANGES_REQUESTED** [review]

**Comment by abhishalya** [comment]
Closing as the PR was abandoned with invalid changes.

**Pull request closed without merging**